src/adagrams.js

```diff
@@ -1,8 +1,127 @@
-const Adagrams = {
-  drawLetters() {
-    // Implement this method for wave 1
-  },
-};
+class Adagrams {
+  static drawLetters() {
+
+    const letterPool = {
```

Comment:
Style note: consider moving `letterPool` outside the function, but inside the module.